tests/test_pretest.py

```diff
@@ -27,7 +29,7 @@
 FEATURE_STATE_VERIFYING_INTERVAL_SECS = 10
 
 
-def test_features_state(duthosts, enum_dut_hostname):
+def test_features_state(duthosts, enum_dut_hostname, localhost):
```

Comment:
this looks like a different pr

Reply:
yes, this part will be merged by #4240.

Reply:
This change still show up but I did a diff against github/master and confirmed that the reboot change is no longer outstanding diff.